diff --git a/Configurations/10-main.conf b/Configurations/10-main.conf
index c9db9aac28..023828c46c 100644
--- a/Configurations/10-main.conf
+++ b/Configurations/10-main.conf
@@ -697,7 +697,7 @@ my %targets = (
     },
     "linux-latomic" => {
         inherit_from     => [ "linux-generic32" ],
-        ex_libs          => add(threads("-latomic")),
+        ex_libs          => add(threads("-pthread")),
     },
     "linux-generic64" => {
         inherit_from     => [ "linux-generic32" ],
